### PR TITLE
Displays current players turn

### PR DESCRIPTION
**Pull request by Cristian-Baeza**
DIsplays the player symbol depending on whos turn it is

### CODE REVIEW

**Review by td-extreme — COMMENTED** [review]
Is there a way that you can figure out who's turn it is without having to store that in a variable?

File: lib/questions.rb [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
-    @player_counter += 1
-  end
+  def turn_count(board)
+    counter = 0
+    board.each do |space|
+       if space == "X" || space == "O"
+          counter += 1
+        end
+    end
+    counter
```
Comment (td-extreme) [inline lib/questions.rb:10]: There should be a way to do this without having to write a for each. Perhaps using the .count method
https://ruby-doc.org/core-2.7.0/Array.html

File: lib/questions.rb [hunk 2]
```diff
@@ -1,29 +1,32 @@
...
-  attr_reader :player_counter
 
-  def initialize()
-    @player_counter = 2
-  end
-
-  def increase_player_counter
-    @player_counter += 1
-  end
+  def turn_count(board)
```
Comment (td-extreme) [inline lib/questions.rb:3]: Your questions class should not be responsible for figuring out the turn_count.

Comment (Cristian-Baeza) [reply lib/questions.rb:3]: ahhh gotcha, I understand what you meant now 👍🏽

File: lib/questions.rb (hunk 1 above)
Comment (td-extreme) [reply lib/questions.rb]: I'm not sure of the difference in performance.
This is more of a readability issue.

Comment (Cristian-Baeza) [reply lib/questions.rb]: 10-4

File: lib/tictactoe.rb [hunk 3]
```diff
@@ -68,6 +68,20 @@ def check_for_winner?(board)
...
 
+  def turn_count(board)
+    counter = 0
+    board.each {|space| 
+      if space == "X" || space == "O"
+          counter += 1
+        end }
+    
+    counter
+ end
```
Comment (td-extreme) [inline lib/tictactoe.rb:79]:
```suggestion
  def turn_count(board)
   9 - board.count(" ")
 end
```

Comment (Cristian-Baeza) [reply lib/tictactoe.rb:79]: Magician

**Pull request merged**